### DISCUSSION TITLE
Make it possible to add a description for the training day

### DISCUSSION THREAD

**Discussion by lsolesen** [astashov/liftosaur#183] [Ideas]
It would be awesome, if it was possible to have a description for the day somehow.

Like what is the purpose of the day, and what should be in the warmup.

![image](https://github.com/user-attachments/assets/0ee08632-cff2-4b7d-8c30-28c7d26bfb43)

 I am not excatly sure what the best markup would be though, but it should obviously be added in Markdown.

![chrome_PSC9IdUM6f](https://github.com/user-attachments/assets/2e870c44-2a2d-4a6a-8046-9af07678eba3)

**Comment by yrinkel** [comment]
This has been implemented now, so this can be closed now?
